### ISSUE TITLE
Broadcom SAI 9.x/10.1 does not allow switch_id when creating fabric switch

### ISSUE THREAD

**Issue by ysmanman** [sonic-net/sonic-buildimage#18579]
#### Description

We observed the following syncd crash on sup with SAI 10.1:

```
Apr  3 00:13:33.568454 nfc407 ERR syncd5#syncd: [1f:00.0] SAI_API_SWITCH:brcm_sai_dnx_create_switch:8075 SAI_SWITCH_ATTR_SWITCH_ID is not valid for SAI_SWITCH_ATTR_TYPE
Apr  3 00:13:33.568689 nfc407 ERR syncd5#syncd: :- sendApiResponse: api SAI_COMMON_API_CREATE failed in syncd mode: SAI_STATUS_INVALID_PARAMETER
Apr  3 00:13:33.569057 nfc407 ERR syncd5#syncd: :- processQuadEvent: attr: SAI_SWITCH_ATTR_INIT_SWITCH: true
Apr  3 00:13:33.569256 nfc407 ERR syncd5#syncd: :- processQuadEvent: attr: SAI_SWITCH_ATTR_FDB_EVENT_NOTIFY: 0x5615c1cf63d0
Apr  3 00:13:33.569450 nfc407 ERR syncd5#syncd: :- processQuadEvent: attr: SAI_SWITCH_ATTR_PORT_STATE_CHANGE_NOTIFY: 0x5615c1cf63e0
Apr  3 00:13:33.569645 nfc407 ERR syncd5#syncd: :- processQuadEvent: attr: SAI_SWITCH_ATTR_SWITCH_SHUTDOWN_REQUEST_NOTIFY: 0x5615c1cf6400
Apr  3 00:13:33.569847 nfc407 ERR syncd5#syncd: :- processQuadEvent: attr: SAI_SWITCH_ATTR_PORT_HOST_TX_READY_NOTIFY: 0x5615c1cf6460
Apr  3 00:13:33.570062 nfc407 ERR syncd5#syncd: :- processQuadEvent: attr: SAI_SWITCH_ATTR_SWITCH_HARDWARE_INFO: 7:49,102,58,48,48,46,48
Apr  3 00:13:33.570281 nfc407 ERR syncd5#syncd: :- processQuadEvent: attr: SAI_SWITCH_ATTR_TYPE: SAI_SWITCH_TYPE_FABRIC
Apr  3 00:13:33.570492 nfc407 ERR syncd5#syncd: :- processQuadEvent: attr: SAI_SWITCH_ATTR_SWITCH_ID: 4294967295
Apr  3 00:13:33.575873 nfc407 ERR swss5#orchagent: :- create: create status: SAI_STATUS_INVALID_PARAMETER
Apr  3 00:13:33.576107 nfc407 ERR swss5#orchagent: :- main: Failed to create a switch, rv:-5
```

According to SAI 10.1 source code, switch_id is not allowed when creating fabric switch. 
Opened CSP CS00012344284 to tack the issue.

#### Steps to reproduce the issue:
1.
2.
3.

#### Output of `show version`:

```
(paste your output here)
```
#### Output of `show techsupport`:

```
(paste your output here or download and attach the file here )
```

**Comment by ysmanman** [comment]
Add @arlakshm @kenneth-arista @arista-nwolfe for viz.

**Comment by arista-nwolfe** [comment]
CS00012344284 is tracking the discussion with Broadcom

**Comment by adyeung** [comment]
Debug and triage will be thru BRCM CSP

**Comment by arlakshm** [comment]
Fix available in DNX SAI version 10.1.15. @arista-nwolfe to validate and close this issue

**Comment by ysmanman** [comment]
This is the SONiC PR that extracts the fabric switch_id from config_db and then passes to SAI: https://github.com/sonic-net/sonic-swss/pull/3102
